src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/Grpc/TrailingHeadersHelpers.cs

```diff
@@ -41,21 +42,15 @@ public static HttpHeaders TrailingHeaders(this HttpResponseMessage responseMessa
         // in RequestMessage.Properties with known key. Return empty collection.
         // Client call will likely fail because it is unable to get a grpc-status.
         return ResponseTrailers.Empty;
+#else
+        return responseMessage.TrailingHeaders;
 #endif
     }
 
 #if NETSTANDARD2_0 || NET462
-    public static void EnsureTrailingHeaders(this HttpResponseMessage responseMessage)
```

Comment:
I verified that this method wasn't being used, which is why it was deleted here.

That confused me while doing my review so just leaving a note for other reviewers :)